commands/local_server_start.go

```diff
@@ -287,11 +287,16 @@ var localServerStartCmd = &console.Command{
 			scheme = "http"
 		}
 
+		address := config.ListenIp
+		if c.Bool("allow-all-ip") {
+			address = "0.0.0.0"
+                }
```

Comment:
tabs instead of spaces:
```suggestion
		if c.Bool("allow-all-ip") {
			address = "0.0.0.0"
		}
```

Reply:
It would make sense to have a CI job checking for code formatting

Reply:
yes! exactly what I had in mind when commenting